test/Elastic.Apm.AspNetCore.Tests/Elastic.Apm.AspNetCore.Tests.csproj

```diff
@@ -7,7 +7,11 @@
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.6.0" />
     <PackageReference Include="FluentAssertions.Analyzers" Version="0.11.4" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0">
```

Comment:
Is there any thinking behind changing `PrivateAssets` and `IncludeAssets`? If yes could we maybe add a comment for those?

Reply:
I actually used the default configuration explained in the docs: [here](https://github.com/tonerdo/coverlet/blob/master/Documentation/VSTestIntegration.md#coverlet-integration-with-vstest-aka-visual-studio-test-platform), so I'm not really familiar with all the setup, how do you wanna proceed? To add a comment with the above link?

Reply:
I see. So with these settings we change how assets flow through referenced projects - I was just wondering if this is needed and why it's not the default. 

E.g. I was wondering why `PrivateAssets` is needed - but given that `Elastic.Apm.Tests` is referenced by other test projects, probably we'd have problems without that setting.

Anyways, this is fine. 

> how do you wanna proceed? 

Let's just merge, no comment needed. If I or someone feels this is important to fully understand we can dig deeper, but it's fine for me now.